File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/commands/low_level/intake/SetIntakeGripStateCommand.java
package org.firstinspires.ftc.teamcode.commands.low_level.intake;

import com.arcrobotics.ftclib.command.InstantCommand;

import org.firstinspires.ftc.teamcode.managers.IntakeManager;

public class SetIntakeGripStateCommand extends InstantCommand {
    public SetIntakeGripStateCommand(IntakeManager manager, IntakeManager._ClawState clawState) {
        super(() -> manager.update(clawState));
    }
}

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/commands/low_level/intake/SetIntakeGripStateCommand.kt
package org.firstinspires.ftc.teamcode.commands.low_level.intake;

import com.arcrobotics.ftclib.command.InstantCommand;

import org.firstinspires.ftc.teamcode.managers.IntakeManager;
import org.firstinspires.ftc.teamcode.managers.IntakeManager._GripState

class SetIntakeGripStateCommand( manager: IntakeManager, _GripState: _GripState) : InstantCommand({manager.update(_GripState)})
